### ISSUE TITLE
Why do you release a pro version ? 

### ISSUE THREAD

**Issue by F3000** [SimpleMobileTools/Simple-Contacts#301]

**Comment by tibbi** [comment]
you can read it at https://medium.com/@tibbi/some-simple-mobile-tools-apps-are-becoming-paid-d053268f0fb2

**Comment by F3000** [comment]
And the github version ? Here I can’t buy something . How do you implement this ?

**Comment by tibbi** [comment]
it is free here, you just have to uninstall the old version and then install a fresh new 6.0, since the signature changed.

**Comment by F3000** [comment]
Ok for understanding : 

Play store variant (v6) one week free . After a week I have to pay. 
GitHub variant is free when  I deinstall the old version befor ? 


Is my understanding right ?

**Comment by tibbi** [comment]
If you download the Google Play version within a week, you will never have to pay for it. Not even if you uninstall it and you will want to install it back someday in the future.
Github variant is free in all cases, you just have to uninstall the old version to be able to get the new one. It is just because the certificate used for signing the app has changed.

**Comment by F3000** [comment]
Thanks for explaining . 👍

**Comment by Silent-Hunter** [comment]
What about the versions published on F-Droid? Will there be a way to pay there?

**Comment by tibbi** [comment]
well paying wont be mandatory there, f-droid apps already contain some donation urls. You can find some on https://simplemobiletools.github.io/donate/ too

**Comment by immerfroehlich** [comment]
The app informed me today, that it will no longer be maintained in the F-Droid Store. Is this correct?

**Comment by tibbi** [comment]
some free apps wont be maintained anymore anywhere. Only those that have a Pro version already. All pro versions will be added on f-droid soon, for free.

**Comment by immerfroehlich** [comment]
Well that's a very generous act of you. I donated a small amount, to help you maintain your great app and maybe add a few features I will ask for in the future. ;-)
Thank you for this great app.

**Comment by Shelly342018** [comment]
Is there a free pro version download for all devices that came with your app? 
I really hope so because my J3 Luna Pro and my tablet came with your app preinstalled and I'm having way too many issues with it!
The main problem is loss of contacts and contact photos.
I tried importing but MMS/link sharing/Bluetooth sharing wouldn't work and when I restored from backup almost all of my contact photos were gone. No default color circle or anything, just gray!
